source/managing_account/03_checking_account_quota/checking_account_quota.rst

```diff
@@ -0,0 +1,8 @@
+Checking account quota
+======================
+
+From the Quota tab of their account, authenticated users have the possibility to check how many API calls they have made compared to how many they are allowed. This allowed quota is defined per user by the portal administrators.
+
+These quotas are daily: they are reset every day at 1:00 AM.
```

Comment:
Why could clarify the timezone (and honestly I am not sure if it's 1AM UTC or Paris ...)